Yoast/javascript
# Add Flesch reading and passive voice assessments for Portuguese

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds Flesch reading ease assessment for Portuguese.
* [yoastseo] Adds passive voice assessment for Portuguese.
* [Yoast SEO Free] Adds Flesch reading ease assessment for Portuguese.
* [Yoast SEO Free] Adds passive voice assessment for Portuguese.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Create a Portuguese text that contains passive and/or active Portuguese sentences. 
* For example, you can use some of those passive sentences:
  * _A decisão será aprovado._
  * _Cada participante foi informado._
  * _A nova regra será integrada._
  * _Todas as regras foram protocoladas durante a reunião._
* Or these active sentences:
  * _Eu aprovo esta decisão._
  * _O líder informou todos os participantes da reunião._
  * _Eles tomaram a decisão quando todos estavam presentes._
  * _É o método preferido de vários membros da equipe de produção._
* Check that the result of the passive voice assessment is in line with the [scoring schema](https://github.com/Yoast/javascript/wiki/Scoring-readability-analysis#5-passive-voice).

* Create a Portuguese text that is difficult to read - it contains long sentences and complex words.
* Make sure that the Flesch Reading assessment appears in the assessment bar and the score is on the lower side (see [scoring schema](https://github.com/Yoast/javascript/wiki/Scoring-readability-analysis#7-flesch-reading-ease)).
* Split longer sentences into shorter ones, remove complex words from the text.
* The score of the Flesch reading score goes up.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-465
